index.js

```diff
@@ -45,7 +46,8 @@ class Plex {
 
   getState(callback) {
     const options = {
-      url: `http://${this.host}:${this.port}/status/sessions`,
+      url: `http${this.secure ? "s" : ""}://${this.host}:${this.port}/status/sessions`,
```

Comment:
Could you please use single quotes instead of doubles here? ESlint should have warned you.